src/nupic/algorithms/AnomalyLikelihood.cpp

```diff
@@ -14,9 +14,9 @@ namespace nupic {
 namespace algorithms {
 namespace anomaly {
 
-   Real compute_mean(const vector<Real>& v); //forward declaration
-   Real compute_var(const vector<Real>& v, Real mean); 
-   static UInt calcSkipRecords_(UInt numIngested, UInt windowSize, UInt learningPeriod);
+static Real compute_mean(const vector<Real>& v); //forward declaration
+static Real compute_var(const vector<Real>& v, Real mean); 
+static UInt calcSkipRecords_(UInt numIngested, UInt windowSize, UInt learningPeriod);
```

Comment:
this (the `static`) does not seem correct. If I instantiate more AnomalyLikelihoods (say 2 regions..) this code will break the functionality.

Reply:
Don't know what the story is.  I had to add static in the forward declaration so that they would compile. They were declared static below where they were actually implemented.

They don't touch local variables so there should be no problem breaking functionality if there are more than one instance.

Reply:
ok not to deal with in in this PR. 

> They don't touch local variables so there should be no problem breaking functionality if there are more than one instance. 

If they're used, it'll break in (multi-threaded?) runtime collision. -> the vars should not be static.